website_sale_product_attribute_value_filter_existing/static/src/js/website_sale_product_attribute_value_filter_existing_tour.esm.js

```diff
@@ -10,24 +10,23 @@ tour.register(
     },
     [
         {
-            content: "1: browse shop page",
-            trigger: "a[href='/shop']",
-        },
-        {
-            content: "2: search a product",
-            trigger: "input[name=search]",
-            run: "text Ipod",
-            extra_trigger: ".js_attributes:has(label:contains('Test blue'))",
+            content: "search a product",
+            trigger: "form input[name=search]",
+            run: "text desk",
+            extra_trigger:
+                ".js_attributes:not(:contains('Test yellow')):has(label:contains('Test red')), .js_attributes:not(:contains('Test yellow')):has(label:contains('Test blue')), .js_attributes:not(:contains('Test yellow')):has(label:contains('Test green'))",
```

Comment:
It would help to document every step, as it's hard to follow from the selector itself

Reply:
done :)